lib/Backends/OpenCL/kernels.cl

```diff
@@ -755,11 +755,12 @@ __kernel void softmaxgradW(__global void *mem, cl_uint32_t origDest,
 __kernel void convolutionK(__global float *dest, __global float *src,
                            __global float *filter, __global float *bias,
                            ShapeHW filterSizes, ShapeHW strides,
-                           PaddingTLBR pads, ShapeNHWC odim, ShapeNHWC idim,
+                           PaddingTLBR pads, cl_uint32_t group, ShapeNHWC odim, ShapeNHWC idim,
                            ShapeNHWC filterDim) {
   size_t ax = get_global_id(0);
   size_t ay = get_global_id(1);
   size_t d = get_global_id(2);
+  size_t offset = d / (odim.c / group) * (idim.c / group);
```

Comment:
nit: offset is general name, which interferes a bit with scale/offset in Quantization case. How does `inChannelOffset` sound?
Also, let's make code more similar to Interpreter implementation by having `inCperG` and `outCperG`.